backends/events/sdl/sdl-events.cpp

```diff
@@ -210,198 +215,383 @@ void SdlEventSource::SDLModToOSystemKeyFlags(SDL_Keymod mod, Common::Event &even
-	case SDLK_g: return Common::KEYCODE_g;
-	case SDLK_h: return Common::KEYCODE_h;
-	case SDLK_i: return Common::KEYCODE_i;
-	case SDLK_j: return Common::KEYCODE_j;
-	case SDLK_k: return Common::KEYCODE_k;
-	case SDLK_l: return Common::KEYCODE_l;
-	case SDLK_m: return Common::KEYCODE_m;
-	case SDLK_n: return Common::KEYCODE_n;
-	case SDLK_o: return Common::KEYCODE_o;
-	case SDLK_p: return Common::KEYCODE_p;
-	case SDLK_q: return Common::KEYCODE_q;
-	case SDLK_r: return Common::KEYCODE_r;
-	case SDLK_s: return Common::KEYCODE_s;
-	case SDLK_t: return Common::KEYCODE_t;
-	case SDLK_u: return Common::KEYCODE_u;
-	case SDLK_v: return Common::KEYCODE_v;
-	case SDLK_w: return Common::KEYCODE_w;
-	case SDLK_x: return Common::KEYCODE_x;
-	case SDLK_y: return Common::KEYCODE_y;
-	case SDLK_z: return Common::KEYCODE_z;
-	case SDLK_DELETE: return Common::KEYCODE_DELETE;
-	case SDLK_KP_PERIOD: return Common::KEYCODE_KP_PERIOD;
-	case SDLK_KP_DIVIDE: return Common::KEYCODE_KP_DIVIDE;
-	case SDLK_KP_MULTIPLY: return Common::KEYCODE_KP_MULTIPLY;
-	case SDLK_KP_MINUS: return Common::KEYCODE_KP_MINUS;
-	case SDLK_KP_PLUS: return Common::KEYCODE_KP_PLUS;
-	case SDLK_KP_ENTER: return Common::KEYCODE_KP_ENTER;
-	case SDLK_KP_EQUALS: return Common::KEYCODE_KP_EQUALS;
-	case SDLK_UP: return Common::KEYCODE_UP;
-	case SDLK_DOWN: return Common::KEYCODE_DOWN;
-	case SDLK_RIGHT: return Common::KEYCODE_RIGHT;
-	case SDLK_LEFT: return Common::KEYCODE_LEFT;
-	case SDLK_INSERT: return Common::KEYCODE_INSERT;
-	case SDLK_HOME: return Common::KEYCODE_HOME;
-	case SDLK_END: return Common::KEYCODE_END;
-	case SDLK_PAGEUP: return Common::KEYCODE_PAGEUP;
-	case SDLK_PAGEDOWN: return Common::KEYCODE_PAGEDOWN;
-	case SDLK_F1: return Common::KEYCODE_F1;
-	case SDLK_F2: return Common::KEYCODE_F2;
-	case SDLK_F3: return Common::KEYCODE_F3;
-	case SDLK_F4: return Common::KEYCODE_F4;
-	case SDLK_F5: return Common::KEYCODE_F5;
-	case SDLK_F6: return Common::KEYCODE_F6;
-	case SDLK_F7: return Common::KEYCODE_F7;
-	case SDLK_F8: return Common::KEYCODE_F8;
-	case SDLK_F9: return Common::KEYCODE_F9;
-	case SDLK_F10: return Common::KEYCODE_F10;
-	case SDLK_F11: return Common::KEYCODE_F11;
-	case SDLK_F12: return Common::KEYCODE_F12;
-	case SDLK_F13: return Common::KEYCODE_F13;
-	case SDLK_F14: return Common::KEYCODE_F14;
-	case SDLK_F15: return Common::KEYCODE_F15;
-	case SDLK_CAPSLOCK: return Common::KEYCODE_CAPSLOCK;
-	case SDLK_RSHIFT: return Common::KEYCODE_RSHIFT;
-	case SDLK_LSHIFT: return Common::KEYCODE_LSHIFT;
-	case SDLK_RCTRL: return Common::KEYCODE_RCTRL;
-	case SDLK_LCTRL: return Common::KEYCODE_LCTRL;
-	case SDLK_RALT: return Common::KEYCODE_RALT;
-	case SDLK_LALT: return Common::KEYCODE_LALT;
-	case SDLK_MODE: return Common::KEYCODE_MODE;
-	case SDLK_HELP: return Common::KEYCODE_HELP;
-	case SDLK_SYSREQ: return Common::KEYCODE_SYSREQ;
-	case SDLK_MENU: return Common::KEYCODE_MENU;
-	case SDLK_POWER: return Common::KEYCODE_POWER;
+	case SDLK_BACKSPACE:
```

Comment:
It looks like your editor, or whatnot, messed with this code formatting.

As you may see, this formatting was done on purpose in order to reduce vertical space taken by the code.

Please, revert this change.

Reply:
Sorry about that, yes I will revert it.